_posts/2015-10-14-Four-days-DDD-course.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+name: When you get more than what you were expecting
+title: When you get more than what you were expecting
+date: 2015-10-14 09:35:00 +00:00
+author: Ana Nogal
+canonical:
+    name: my personal blog
+    href: http://www.ananogal.com/blog/when-you-get-more-than-what-you-were-expecting/
+image:
+    src: /assets/img/custom/blog/DDD_SkillsMatter.jpg
+tags:
+- DDD
+- Learning
+-
+---
+Almost 2 weeks ago, I attended a course at [SkillsMatter](http://skillsmatter.com/) about DDD. The first 2 days were intensive and a reinforce of a lot of concepts that I already had and learning a few more.
+
+I really loved some analogies that [Alberto](https://twitter.com/ziobrando) showed us: like the one of the top gun game where he explain to us that our domain is a plain and that when you want to make a shot you have to make the sight aim to the target. That’s what our model should do. And if you can reduce that delta, then you have the perfect model for your domain.
```

Comment:
I really loved some of the analogies that [Alberto](https://twitter.com/ziobrando) showed us. My favourite was the  'Top Gun game' analogy. Alberto explained that our domain is a plane. When you want to make a shot you have to make the sight aim to the target. That’s what our model should do. And if you can reduce that delta, then you have the perfect model for your domain.